packages/xo-cli/index.mjs

```diff
@@ -422,9 +425,18 @@ async function createToken(args) {
 COMMANDS.createToken = createToken
 
 async function register(args) {
-  const opts = await parseRegisterArgs(args, 'xo-cli --register', true)
+  let { clientId } = await config.load()
+  if (clientId === undefined) {
+    clientId = Math.random().toString(36).slice(2)
+  }
+
+  const { name, version } = PACKAGE_JSON
+  const label = `${name}@${version} - ${os.hostname()} - ${os.type()} ${os.machine()}`
+
+  const opts = await parseRegisterArgs(args, label, { id: clientId }, true)
```

Comment:
IMO, it will be useful to add a label to this token, to know that it is an `xo-cli` token.
```suggestion
  const opts = await parseRegisterArgs(args, label, { id: clientId, label: "XO-CLI token" }, true)
```